src/main/java/com/minecolonies/coremod/entity/ai/citizen/lumberjack/Tree.java

```diff
@@ -371,6 +382,48 @@ private void addAndSearch(@NotNull final World world, @NotNull final BlockPos lo
     }
 
     /**
+     * Adds a leaf and searches for further leaves.
+     *
+     * @param world The world the leaf is in.
+     */
+    private void addAndSearch(@NotNull final World world)
+    {
+        int locXMin = location.getX() - 4;
+        int locXMax = location.getX() + 4;
+        int locYMin = location.getY() + 2;
+        int locYMax = MAX_TREE_SIZE;
+        int locZMin = location.getZ() - 4;
+        int locZMax = location.getZ() + 4;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Assign this magic number 4 to a well-named constant, and use the constant instead. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AS109)